apps/mull-ui/src/app/components/navigation/sub-nav-bar/chat-header.tsx

```diff
@@ -4,13 +4,30 @@ import './chat-header.scss';
 export interface ChatHeaderProps {
   style?: CSSProperties;
   className?: string;
-  eventTitle: string;
+  isDirectMessage?: boolean;
+  userAvatarUrl?: string;
+  userName?: string;
+  eventTitle?: string;
 }
 
-export const ChatHeader = ({ style, className, eventTitle }: ChatHeaderProps) => {
+export const ChatHeader = ({
+  style,
+  className,
+  eventTitle,
+  isDirectMessage,
+  userAvatarUrl,
+  userName,
+}: ChatHeaderProps) => {
   return (
-    <div className={`chat-header ${className ? className : ''}`} style={style}>
-      <h1 className="chat-header-title">{eventTitle}</h1>
+    <div className={`chat-header top-nav-bar-shadow ${className ? className : ''}`} style={style}>
+      {isDirectMessage ? (
+        <div className="direct-message-header-title" data-testid="direct-message-header">
+          <img className="direct-message-header-image" src={userAvatarUrl} alt="user" />
+          <h2 className="direct-message-header-name">{userName}</h2>
```

Comment:
We have 64 characters max for the user's display name, should we also have the username end with `...` past a certain character count?
![image](https://user-images.githubusercontent.com/33991121/113948267-75cd8500-97da-11eb-98ab-27a5f8176295.png)

Reply:
Same as before, you should be able to fix this easily with styling:
https://www.w3schools.com/cssref/css3_pr_text-overflow.asp